yellowbrick/features/base.py

```diff
@@ -190,7 +212,11 @@ def __init__(self, ax=None, features=None, classes=None, color=None,
         # Visual Parameters
         self.color = color
         self.colormap = colormap
-
+        try:
+            # Ensures that taregt is either Single, Discrete, Continuous or Auto
+            self.target = target_type(target)
+        except:
```

Comment:
What kind of exception does this raise? 

Also what happens when you do `target_type(target_type.AUTO)`?

Reply:
Also, small typo in the comment ;-)

Reply:
It raises`ValueError`. 

`target_type(target_type.AUTO)`  and 'target_type("AUTO")` have same results.

I will correct the typo.

Reply:
per your above comment, would you mind making this `except ValueError:`?

Reply:
Awesome - if so, perhaps we should have the default be `TargetType.AUTO` instead of the string?

Reply:
We certainly can do so. But I think that it may confuse the user because a string type variable would have defaulted to some Enum. We can go with it if such examples are common.

Reply:
It doesn't really matter; let's leave it as a string for now.